vsphere_plugin_common/__init__.py

```diff
@@ -1394,19 +1394,24 @@ def _validate_windows_properties(
 
     def _add_network(self, network, datacenter):
         network_name = network['name']
+        normalised_network_name = self._get_normalised_name(network_name).lower()
```

Comment:
Shouldn't the .lower() be done as part of normalising the network name?

Maybe as a defaulted argument if you also need _get_normalised_name to be able to provide case sensitive checks.

Reply:
Fixed in 999e3694ee1aa8d6c574cd263716c0dab022d418 - plugin compare lowered version.